main.go

```diff
@@ -127,9 +127,61 @@ func messageCreate(s *discordgo.Session, m *discordgo.MessageCreate) {
 		return
 	}
 
-	// Check if it's a command we recognize
+	if strings.HasPrefix(m.Content, "!skill") {
+		//TODO: Finish this command
+		//cmdSkill(m.Content, channel, s, m)
+	}
+
 	if strings.HasPrefix(m.Content, "!leaguewatch") {
-		tokens := strings.Split(m.Content, " ")
+		cmdLeagueWatch(m.Content, channel, s, m)
+	}
+
+	if strings.HasPrefix(m.Content, "!help") {
+		cmdHelp(channel, s, m)
+	}
+
+
+}
+
+func cmdSkill(c string, c2 *discordgo.Channel, s *discordgo.Session, m *discordgo.MessageCreate) {
+	//TODO: Finish this command
+	// Check if it's a command we recognize
+	if strings.HasPrefix(c, "!skill") {
+		tokens := strings.Split(c, " ")
+		if len(tokens) != 3 {
+			// Unrecognized, must be in the form: "!skill champ q
+			log.Debug("Ignoring a unrecognized command: %s", m.Content)
```

Comment:
A helpful feature of logrus is to put dynamic content in fields, then have standardized error messages, the API for that looks like 
```go
log.WithField("command", m.Content).Warn("ignoring an unrecognized command")
```
and allows you to use log analysis tools to both group "unrecognized command" errors, and also still group/filter on the `"command"` if you want